config_quick_debug.yml

```diff
@@ -28,4 +28,26 @@ scenario_params_list:
     - 2
    minibatch_count:
     - 2
-
+ - dataset_name:
+    'titanic':
+    - 'random_initialization'
+   partners_count:
+    - 3
+   amounts_per_partner:
+    - [0.2, 0.5, 0.3]
+   samples_split_option:
+    - ['basic', 'random']
+   multi_partner_learning_approach:
+    - 'fedavg'
+   aggregation_weighting:
+    - 'uniform'
+   epoch_count:
+    - 2
+   methods:
+    - ["Shapley values", "Independent scores"]
+   minibatch_count:
+    - 2
+   gradient_updates_per_pass_count:
+    - 2
+   is_quick_demo:
+    - True
```

Comment:
We don't want to overcharge the quick debug. So consider removing this part. 
However you can add an end-to-end like this one --> https://github.com/SubstraFoundation/distributed-learning-contributivity/blob/master/tests/end_to_end_tests.py